EXPLAINER.md

```diff
@@ -0,0 +1,227 @@
+
+A “manifest” is a list of the passengers or cargo on a ship. For web publications, a manifest lists the constituents of the publication—all the HTML files, stylesheets, images, scripts, etc.—needed to create the whole. It further describes the sequence of primary resources, so that we know that chapter-02.html comes after chapter-01.html. In EPUB we called this list the "spine"; for web publications it's now the `readingOrder`. 
+
+The manifest is also the natural location for metadata that applies to the whole publication, rather than just one of the constituents. The metadata vocabulary is based on schema.org; the entire manifest is serialized as JSON-LD. 
+
+
+## Example Manifest
+
+Here's a simple example of a web publication manifest, for a tiny version of *Moby-Dick* with only a few HTML files:
+
+```json
+{
+    "@context": ["https://schema.org", "https://www.w3.org/ns/wp-context"],
+    "type": "Book",
+    "url": "https://publisher.example.org/mobydick",
+    "author": "Herman Melville",
+    "dateModified": "2018-02-10T17:00:00Z",
+
+    "readingOrder": [
+        "html/title.html",
+        "html/copyright.html",
+        "html/introduction.html",
+        "html/epigraph.html",
+        "html/c001.html",
+        "html/c002.html",
+        "html/c003.html",
+        "html/c004.html",
+        "html/c005.html",
+        "html/c006.html"
+    ],
+
+    "resources": [
+        "css/mobydick.css",
+        {
+            "type": "PublicationLink",
+            "rel": "https://www.w3.org/ns/wp#cover-page",
+            "url": "images/cover.jpg",
+            "encodingFormat": "image/jpeg"
+        },{
+            "type": "PublicationLink",
+            "url": "html/toc.html",
+            "rel": "contents"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneral.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralBol.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralBolIta.otf",
+            "encodingFormat": "font/otf"
+        },{
+            "type": "PublicationLink",
+            "url": "fonts/STIXGeneralItalic.otf",
+            "encodingFormat": "font/otf"
+        }
+    ]
+}
+
+```
+
+Note that `readingOrder` defines the sequence of primary resources that form the publication. `resources` enumerates all the other resources that are required to render the publication. 
+
+## Design choices
+
+The unanswered question:
+
+> is the goal of this group to make a new packaging format for specialist book reading software and devices, or is it to obtain first class support for missing book-related features in the web platform as a whole?
+
+
```

Comment:
I think the quote from Andrew Betts is referring to more than just packaging, but the whole direction of the working group. We've certainly spent much of our time on a high-level data model, rather than identifying low-level primitives that would allow us to more easily build publications on the web. For good or for bad, what we are actually building can most accurately be described as unpackaged EPUB with the OPF serialized as JSON-LD.  

Packaging should and will have a great influence on what happens with WPUB. For example, web packages right now explicitly depend on a manifest, and actually refer to ours as well as WAM. Having information in our manifest that would enable packaging would be very helpful. I hope we avoid a situation where most every publication needs a separate web publication manifest, a web application manifest, and a web packaging manifest.

Reply:
Dave, I do not think we disagree; this shows, however, that just having this question thrown into the explainer without further explanation may not be the right thing to do. That is what motivated my original comment.

Reply:
Ivan, I've moved that question to the "unanswered questions" section, included more of the quote, and tried to provide some context in 312cd04.

Reply:
I am still a bit wary with this question, but I certainly prefer as part of the unanswered questions. Let us do this for now, and see what others comment...